### PR TITLE
Ports BoH bombing rework II

### PR DESCRIPTION
**Pull request by VictorVow**
https://github.com/tgstation/tgstation/pull/45174

## About The Pull Request

Ever since the big BoH nerf, BoH bombing is kind of a joke. The damage it does is not really huge, and it's lame for traitors - it lacks emotion and their target may not even die, as they can use the ladders to get away.

So, what this does is add back some soul into BoH bombing, while not making it too round-ending and adding some degree of damage control for admins in case of griefing.
When BoH bombing, a stationary singulo with custom sprite will spawn. Its gravitational pull is REALLY strong, so people near it are probably going to get eaten, and there will be some real damage to the hull. The singulo will cease existing after 5 seconds.

For antags, I think this becomes much more interesting than the current BoH bombing. In the case of griefers, the damage to the station isn't too big because the singulo is stationary and fades away after some seconds. And then, admins can click a button in their chat to bring back everything the singulo ate into the game (the button expires in 10 minutes, however, but this can be changed if you think it is needed).

Settings like gravitational pull can be tweaked if yall find it to be too strong or something.
## Changelog

:cl:
tweak: BoH bombing changed again. Now it's more violent.
/:cl:

### CODE REVIEW

**Review by ike709 — CHANGES_REQUESTED** [review]
Fails travis.

File: code/game/area/areas/centcom.dm [hunk 1]
```diff
@@ -138,14 +138,6 @@
 	name = "Syndicate Elite Squad"
 	icon_state = "syndie-elite"
 
-/area/fabric_of_reality
```
Comment (MarkSuckerberg) [inline code/game/area/areas/centcom.dm:141]: If you delete this, make sure to delete it from the map.

File: code/game/objects/structures/ladders.dm [hunk 2]
```diff
@@ -219,51 +219,3 @@
 				break  // break if both our connections are filled
 
 	update_icon()
-
-/obj/structure/ladder/unbreakable/binary
```
Comment (MarkSuckerberg) [inline code/game/objects/structures/ladders.dm:223]: I don't think all of these ladders should be removed.

File: code/game/turfs/simulated/chasm.dm [hunk 3]
```diff
@@ -100,19 +100,3 @@
 	underlay_appearance.icon = 'icons/turf/floors.dmi'
 	underlay_appearance.icon_state = "dirt"
 	return TRUE
-
-//For Bag of Holding Bombs
-
-/turf/open/chasm/magic
```
Comment (MarkSuckerberg) [inline code/game/turfs/simulated/chasm.dm:106]: Need to remove this from maps too.

File: code/modules/power/singularity/boh_tear.dm [hunk 4]
```diff
@@ -0,0 +1,72 @@
+/// BoH tear
+/// The BoH tear is a stationary singularity with a really high gravitational pull, which collapses briefly after being created
+/// The BoH isn't deleted for 10 minutes (only moved to nullspace) so that admins may retrieve the things back in case of a grief
+/obj/singularity/boh_tear
+	name = "tear in the fabric of reality"
+	desc = "Your own comprehension of reality starts bending as you stare this."
```
Comment (MarkSuckerberg) [inline code/modules/power/singularity/boh_tear.dm:6]: Change to "...stare at it."

File: tgstation.dme [hunk 5]
```diff
@@ -0,0 +1,2889 @@
+// DM Environment file for tgstation.dme.
```
Comment (MarkSuckerberg) [inline tgstation.dme:1]: This isn't tgstation. Please modify beestation.dme.

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
Some things I noticed

**Comment by VictorVow** [comment]
I'll reopen at a later date when I can make these changes.

**Pull request closed without merging**